lib/Installation/Partitioner/ExpertPartitionerPage.pm

```diff
@@ -109,4 +115,31 @@ sub press_accept_button {
     send_key('alt-a');
 }
 
+sub open_clone_partition_dialog {
+    my ($self) = @_;
+    assert_screen(EXPERT_PARTITIONER_PAGE);
+    send_key($self->{partition_table_shortcut});
+    for (1 .. 2) { wait_screen_change { send_key "down" } }
+    save_screenshot;
+    send_key "ret";
+}
+
+sub select_all_disks_to_clone {
+    my ($self, $numdisks) = @_;
+    assert_screen(CLONE_PARTITION);
+    send_key($self->{available_target_disks_shortcurt});
+    for (1 .. $numdisks) {
+        send_key "spc";
+        wait_screen_change { send_key "down" }
```

Comment:
maybe put an extra condition here to avoid the additional  key call. maybe unless $_==$numdisks

Reply:
I may do this on monday. Though if somebody wants to merge before, I'd also be ok with it.

Reply:
donD